specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -0,0 +1,5736 @@
+      "get": {
+        "tags": [
+          "Script"
+        ],
+        "summary": "Get a Script's revision in a Workspace.",
+        "description": "Get a Script's revision in a Workspace",
+        "operationId": "ScriptRevisions_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/workspaceName"
+          },
+          {
+            "name": "scriptName",
+            "in": "path",
+            "description": "The name of the script.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "revision",
+            "in": "path",
+            "description": "The revision of the script.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Script"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Microsoft.DataCollaboration resource provider.",
+            "schema": {
+              "$ref": "#/definitions/DataShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ScriptRevisions_Get": {
+            "$ref": "./examples/ScriptRevisions_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ADLSGen2FileDataSet": {
+      "description": "An ADLS Gen 2 file data set.",
+      "required": [
+        "properties",
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataSet"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ADLSGen2FileProperties",
+          "description": "AdlsGen2File data source properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "AdlsGen2File"
+    },
+    "ADLSGen2FileProperties": {
+      "description": "Properties of the ADLS Gen2 file data set.",
+      "required": [
+        "fileSystem",
+        "filePath",
+        "subscriptionId",
+        "resourceGroup",
+        "storageAccountName"
+      ],
+      "type": "object",
+      "properties": {
+        "dataSetId": {
+          "description": "Unique id for identifying a data set resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "filePath": {
+          "description": "File path within the file system.",
+          "type": "string"
+        },
+        "fileSystem": {
+          "description": "File system to which the file belongs.",
+          "type": "string"
+        },
+        "resourceGroup": {
```

Comment:
this should take the storage account resource ID, not the components of the ID (applies anywhere this is being done for an azure resource)